src/node_jsvmapi.cc

```diff
@@ -1002,28 +1002,29 @@ napi_status napi_create_array_with_length(napi_env e, int length, napi_value* re
   return GET_RETURN_STATUS();
 }
 
-napi_status napi_create_string(napi_env e, const char* s, napi_value* result) {
+napi_status napi_create_string_utf8(napi_env e, const char* s, size_t length,
```

Comment:
Changed back to `int`, and made `int` vs `size_t` usage more consistent everywhere.